test/rules/invalid_case_patterns_test.dart

```diff
@@ -133,6 +141,7 @@ void f(Object o) {
     ]);
   }
 
+  @FailingTest(reason: 'lint no longer produced with analyzer 5.8.0')
```

Comment:
Should it?

Reply:
No idea. I've filed https://github.com/dart-lang/linter/issues/4166.